### ISSUE TITLE
No icons for apps are shown in the upper right corner

### ISSUE THREAD

**Issue by abetkin** [pop-os/pop#772]
**Steps to reproduce:**
 Install some messagers, like Slack, Telegram, Skype - apps that are always running unless explicitly asked to quit

**Expected behavior:**
An icon of the app is shown in the notification area - the upper right corner. Usually it's used to show the number of notifications (unread messages)

**Actual behavior:**
No icons are shown for Slack, Telegram or Skype 

![telegram](https://user-images.githubusercontent.com/2223279/70000987-33a08d80-156d-11ea-9452-731e9b0cbf2e.png)

**Comment by mickare** [comment]
Hi, for some reasons Pop!_OS is missing the [gnome-shell-extension-appindicator](https://github.com/ubuntu/gnome-shell-extension-appindicator)...

Just follow the Install instructions...

**Comment by abetkin** [comment]
Yes, that is the reason. Pop-os team, feel free to close the bug. I am not doing it now, since it is probably a good idea to install this extension by default.
